src/Phinx/Config/Config.php

```diff
@@ -239,15 +239,18 @@ public function getMigrationPath()
     /**
      * Gets the base class name for migrations.
      *
+     * @param boolean $dropNamespace Return the base migration class name without the namespace.
      * @return string
      */
-    public function getMigrationBaseClassName()
+    public function getMigrationBaseClassName($dropNamespace = true)
     {
         if (!isset($this->values['migration_base_class'])) {
-            return 'AbstractMigration';
+            $className = 'Phinx\Migration\AbstractMigration';
+        } else {
+            $className = $this->values['migration_base_class'];
         }
 
-        return $this->values['migration_base_class'];
+        return $dropNamespace ? basename(str_replace('\\', DIRECTORY_SEPARATOR, $className)) : $className;
```

Comment:
I develop on Windows and realised that it was inconsistent for non Windows. Will use string manipulation.